v21.1/create-changefeed.md

```diff
@@ -172,9 +170,40 @@ Kafka has the following topic limitations:
 - Topics with a period (`.`) and underscore (`_`) can collide on internal Kafka data structures, so you should use either but not both.
 - Characters not accepted by Kafka will be automatically encoded as unicode characters by CockroachDB.
 
-### Advanced Configuration
+## Advanced changefeed configuration
+
+{{site.data.alerts.callout_danger}}
+The configurations and settings explained in these sections will have significant impact on a changefeed's behavior.
+{{site.data.alerts.end}}
+
+The following sections describe settings, configurations, and details to tune changefeeds for particular use cases:
+
+* [High durability delivery](#tuning-for-high-durability-delivery)
+* [High throughput](#tuning-for-high-throughput)
+* [Kafka sinks](#kafka-sink-configuration)
+
+### Tuning for high durability delivery
 
-#### Kafka Sink Configuration
+When designing a system that relies on high durability of message delivery — that is, not missing any message acknowledgement at the downstream sink — consider the following settings and configuration. Before tuning these settings we recommend reading details on our [changefeed at-least-once-delivery guarantee](stream-data-out-of-cockroachdb-using-changefeeds.html#ordering-guarantees).
+
+* Increase the number of seconds before [garbage collection](architecture/storage-layer.html#garbage-collection) with the [`gc.ttlseconds`](configure-replication-zones.html#gc-ttlseconds) setting to provide a higher recoverability window for data if a changefeed fails. For example, if a sink is unavailable, changes are queued until the sink is available again. While the sink is unavailable, changes will be retried until the garbage collection window is reached and then the data is removed.
+  * You can also use the [`protect_data_from_gc_on_pause`](#protect-pause) option to protect the changes from garbage collection.
+* Determine what a successful write to Kafka is with the [`kafka_sink_config: {"RequiredAcks": "ALL"}`](#kafka-required-acks) option, which provides the highest consistency level.
+* Set the [`changefeed.memory.per_changefeed_limit`](cluster-settings.html) cluster setting to a higher limit to give more memory for buffering per single changefeed. Increasing this limit is useful to avoid errors in situations where the sink is unreachable and the memory buffer fills up.
+* Use [Kafka](#kafka) or [cloud storage](#cloud-storage-sink) sinks when tuning for high durability in changefeeds.
+* Ensure data accuracy downstream by using the [`schema_change_events`](#schema-events) and [`schema_schange_policy`](#schema-policy) options. For example, setting `schema_change_events=column_changes` and `schema_change_policy=stop` will trigger an error to the `cockroach.log` file on a [schema change](stream-data-out-of-cockroachdb-using-changefeeds.html#schema-changes-with-column-backfill).
```

Comment:
Hmm it's not really that data is not accurate when a schema change happens, its just that downstream systems may not be set up to ingest the new format post schema change

Reply:
I would also say the changefeed will fail on `schema_change_policy=stop`, that isn't obvious from this paragraph